deploy/dev.yml

```diff
@@ -37,7 +37,7 @@ subjects:
     name: telegraf-operator
     namespace: telegraf-operator
 ---
-apiVersion: admissionregistration.k8s.io/v1beta1
+apiVersion: admissionregistration.k8s.io/v1
```

Comment:
Hi! We will need to include a couple more properties to the webhooks `telegraf.influxdata.com`. When switching to `admissionregistration.k8s.io/v1` it's required that the webhook has an `admissionReviewVersions: ["v1"]` and `sideEffects: None` defined. Since the webhook only understands "v1" this would be the only `admissionReviewVersions` to add.

Reply:
@BondAnthony Sorry about that, missed this part even though I've added it into [helm PR](https://github.com/influxdata/helm-charts/pull/319/files). Thanks for the review!